src/develop/pixelpipe_hb.c

```diff
@@ -2708,8 +2708,12 @@ gboolean dt_dev_pixelpipe_process(
                "[opencl] frequent opencl errors encountered; disabling"
                " opencl for this session!\n");
       dt_control_log
-        (_("darktable discovered problems with your OpenCL setup;"
-           " disabling OpenCL for this session!"));
+        (_("frequent opencl errors encountered; disabling OpenCL for this session!"
```

Comment:
I'll remove frequent here, we don't really know if this is frequent in this case and:

`OpenCL errors encountered; disabling it for this session!`

Seems clear enough to me.

Reply:
right.